models/bamf_nnunet_ct_lungnodules/config/default.yml

```diff
@@ -0,0 +1,36 @@
+general:
+  data_base_dir: /app/data
+  version: 1.0
+  description: default configuration for 3D semantic image segmentation of the lung and lung nodules from ct scan
```

Comment:
Ideally add `(dicom to dicom)` to the description.

Reply:
I have added the text to the description